compensations/src/test/java/org/jboss/narayana/compensations/functional/compensationScoped/MyCompensationHandler.java

```diff
@@ -43,6 +43,8 @@ public void compensate() {
 
         if (myData != null && myData.getData() != null && myData.getData().equals(expectedData)) {
             dataAvailable = true;
+        } else {
+            dataAvailable = false;
```

Comment:
just checking, is this bit required due to the changes? I saw the test change so wondered about exisiting client compatibility. Same question for the other data|Available = false changes